biohansel/subtype/qc/checks.py

```diff
@@ -16,8 +16,8 @@ def is_overall_coverage_low(st: Subtype, df: pd.DataFrame, p: SubtypingParams) -
             or not st.is_fastq_input():
```

Comment:
Perhaps this above statement can be broken down into separate "if" statements without it being all in one?

Reply:
Yeah, that condition wasn't very clear the way it was written, so I flipped the conditions to check for AND of positive things rather than OR of negatives. Also added a unit test for the function in 98cd34d225f47d07f0121400a0f550937a591fa9